### ISSUE TITLE
paddleseg2.0配置项里面所有的选项哪里能查到？

### ISSUE THREAD

**Issue by bbbxixixixi** [PaddlePaddle/PaddleSeg#842]
比如损失函数这一块
loss
损失函数
参数
types : 损失函数列表
type : 损失函数类型，所支持值请参考损失函数库
coef : 对应损失函数列表的系数列表

我看了paddleseg\models\losses路径下只有6钟，lovasz softmax loss不能使用了吗

还有
model
待训练模型
参数
type : 模型类型，所支持值请参考模型库
others : 请参考对应模型训练配置文件

里面说的模型库，指的是首页钟model zoo里面的？backbone之类的配置如果不在model zoo配置里面的只能去paddleseg\models\backbones路径下自己找了？

**Comment by wuyefeilin** [comment]
你好，如果要查看所支持的模型，backbone，可以查看我们的api文档。

lovasz softmax loss 很快就会支持的

**Comment by bit-scientist** [comment]
@wuyefeilin, hi, if the google translation is correct , the issue is about all options for cfg, right? 😃 

I could see that cfg files has different formats. For example:

```
loss: #version 1
  types:
    - type: CrossEntropyLoss
      weight: [1.013, 1.0, 1.727, 4.085, 3.009, 3.009]
    - type: LovaszSoftmaxLoss
  -coef: [0.5, 0.5]

loss: #version 2
  types:
    - type: MixedLoss
      losses:
        - type: CrossEntropyLoss
          weight: [1.013, 1.0, 1.727, 4.085, 3.009, 3.009]
        - type: LovaszSoftmaxLoss
      coef: [0.5, 0.5]
  coef: [1]
```
How may I know which fomrat is true if I am going to use cityscapes' pretrained model?
Does the the format of the cfg change based solely on what pre-trained weights (hence algorithm) I choose? Thank you!

**Comment by Asthestarsfalll** [comment]
> Does the the format of the cfg change based solely on what pre-trained weights (hence algorithm) I choose? Thank you!

whatever the dataset you use, the format is same.

The version 1 requires that output of your model has 2 elements. The `CrossEntropyLoss` will be applied to the first output, and the `LovaszSoftmaxLoss` will be applied to the second output

The version 2 requires that only 1 element in output, both losses will be applied to it.

**Comment by bit-scientist** [comment]
awesome, @Asthestarsfalll, thanks a lot
